config/local_devnet/base_sp_rollup.json

```diff
@@ -1,19 +1,18 @@
 {
     "genesis": {
-	"l1": {
-	    "hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-	    "number": 0
-	},
-	"l2": {
-	    "hash": "0x85d973858306241634c3a5ecbe114a0754be9d25da523cdd7e1968a09e419a32",
-	    "number": 0
-	},
-	"system_config": {
-	    "batcherAddr": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-	    "overhead": "0x0000000000000000000000000000000000000000000000000000000000000000",
-	    "scalar": "0x0000000000000000000000000000000000000000000000000000000000000000",
-	    "gasLimit": 30000000
-	}
+        "l1": {
+            "hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "number": 0
+        },
+        "l2": {
+            "number": 0
+        },
+        "system_config": {
+            "batcherAddr": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+            "overhead": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
I guess the idea is to set theses values here and then write them to `L1Oracle` in `create_l1_oracle_update_transaction` right?